src/change_stream/mod.rs

```diff
@@ -0,0 +1,134 @@
+//! Contains the functionality for ChangeStreams.
+pub mod document;
+pub mod option;
```

Comment:
For consistency with our other options modules, let's rename this to `options` rather than `option`, then mark it as `pub(crate)` and update [this import](https://github.com/mongodb/mongo-rust-driver/blob/2e7ce917827bdcc19cb7fe18e2d7a61b3afa687c/src/options.rs#L18-L25) to re-export these types as well.

Reply:
fixed